lhotse/recipes/commonvoice.py

```diff
@@ -263,6 +263,11 @@ def parse_utterance(
         raise ValueError(f"No such file: {audio_path}")
     recording_id = Path(row.path).stem
     recording = Recording.from_file(audio_path, recording_id=recording_id)
+    # Handling accent(s) in different versions of CommonVoice
+    if "accents" in row:
```

Comment:
I usually see people using `hasattr(row, 'accents')`.
The operator `in` is commonly used together with `list`, `set` and`dict`.

Not sure whether using the operator `in` is appropriate here.

Reply:
AFAIK `in` checks for column existence in pandas objects, so it looks OK to me.

Reply:
Actually `hasattr` might not work, I think pandas "columns-as-members" thing works only dynamically via `getattr` (unless they explicitly added support for `hasattr`)

Reply:
Ah, thanks. I was not aware that it is a pandas object.